deployment/README.md

```diff
@@ -0,0 +1,265 @@
+* `IMAGE_VERSION`: Specify version for logging component images; e.g. for "openshift/origin-logging-deployer:v1.1", set version "v1.1"
+* `ES_INSTANCE_RAM`: Amount of RAM to reserve per ElasticSearch instance.
+* `ES_CLUSTER_SIZE`: How many instances of ElasticSearch to deploy. At least 3 are needed for redundancy, and more can be used for scaling.
+* `ENABLE_OPS_CLUSTER`: If "true", configure a second ES cluster and Kibana for ops logs.
+* `KIBANA_OPS_HOSTNAME`, `ES_OPS_INSTANCE_RAM`, `ES_OPS_CLUSTER_SIZE`: Parallel parameters for the ops log cluster.
+
+With example parameters:
+
+    oc process logging-deployer-template \
+               -v KIBANA_HOSTNAME=kibana.example.com,PUBLIC_MASTER_URL=https://localhost:8443 \
+               | oc create -f -
+
+Check the logs of the resulting pod (`oc logs <pod name>`) for some instructions
+to follow after deployment. More details are given below.
+
+## Deploy the templates created by the deployer
+
+### Supporting definitions
+
+Create the supporting definitions (you must be cluster admin):
+
+    oc process logging-support-template | oc create -f -
+
+Enable fluentd service account - edit SCC with the following
+
+    oc edit scc/privileged
+
+Add one line as the user at the end: (note, change `:logging:` below to the project of your choice)
+
+    - system:serviceaccount:logging:aggregated-logging-fluentd
+
+Give the account access to read labels from all pods:
+
+    openshift admin policy add-cluster-role-to-user cluster-reader system:serviceaccount:logging:aggregated-logging-fluentd
+
+
+### ElasticSearch
+
+Scaling a deployment today assumes that all pods can safely share
+any volumes specified in the deployment. This is not the case with
+ElasticSearch; each pod requires its own storage. Work is under way
+to enable specifying multiple volumes to be allocated to instances in
+a deployment, but for now multiple deployments are used in order to
+scale ElasticSearch. You can view them all with:
+
+    oc get dc --selector logging-infra=elasticsearch
+
+It is possible to scale your cluster up after creation by adding more
+deployments; however, when scaling up (or down), you must be aware of
+cluster parameters that vary by cluster size and adjust them accordingly
+for both new and existing deployments. Elastic [discusses these issues
+here](https://www.elastic.co/guide/en/elasticsearch/guide/current/_important_configuration_changes.html)
+and the corresponding parameters are coded into the deployments and the
+template below.
+
+The deployer defines a template which it uses to create ElasticSearch
+deployments. You can adjust and reuse it to add more:
+
+    oc process logging-es-template | oc create -f -
+
+These deployments all have different names but will cluster with each other
+via `service/logging-es-cluster`.
+
+Refer to [Elastic's
+documentation](https://www.elastic.co/guide/en/elasticsearch/guide/current/hardware.html#_disks)
```

Comment:
Same here?  Might they have more relaxed requirements for theolder version of ES